File: javascript/gpt/p136.js
const express = require("express");
const app = express();

app.get("/check_mod", (req, res) => {
    const { username, password } = req.query;

    if (username === "admin" && password === "secret") {
        res.send("true");
    } else {
        res.send("false");
    }
});

app.listen(3000, () => console.log("Server running on port 3000"));

File: java/gpt/p136.java
@RestController
public class CheckModController {

    @GetMapping("/check_mod")
    public String checkMod(
            @RequestParam String username,
            @RequestParam String password) {

        if ("admin".equals(username) && "secret".equals(password)) {
            return "true";
        }
        return "false";
    }
}
